DeviceWhoAmI.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$id": "http://https://github.com/harp-tech/protocol/DeviceWhoAmI.json",
```

Comment:
I would use lowercase for the schema name to fit the style of a URL and also to be consistent with the device schema names.

For example, right now we have the following schema ids:
  - https://harp-tech.org/2023-01/device
  - https://harp-tech.org/2023-02/ios
  - https://harp-tech.org/2023-03/registers

The timestamp is useful to know date of publication, for consistency I would suggest maybe the following id:
  - https://harp-tech.org/2023-07/whoami

Which means the filename would be simply `whoami.json`